README.md

```diff
@@ -68,6 +68,25 @@ ng serve -o
 
 This will start a local server which will watch for file changes and reload for you. The port will be displayed to you as `http://localhost:3000`.
 
+## Logging in/Accessing the App
+
+The easiest way to start using the application is to update the contents of the environment.ts file to the following:
+
```

Comment:
can you change this comment to using the appropriate package.json target e.g. npm run build:qa which will build the app with the QA environment settings?